common/src/main/java/com/microsoft/identity/common/internal/cache/MicrosoftStsAccountCredentialAdapter.java

```diff
@@ -208,19 +205,8 @@ public RefreshToken asRefreshToken(com.microsoft.identity.common.internal.provid
         refreshTokenOut.setTarget(refreshTokenIn.getTarget());
         refreshTokenOut.setCachedAt(String.valueOf(TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis())));
         refreshTokenOut.setExpiresOn(refreshTokenIn.getExpiresOn());
```

Comment:
expires on should not apply to refresh tokens.

Reply:
Resolved via 69c1801b80e629b14b639f31d45620b860ca81cc